send/ad_group_mu_ucn

```diff
@@ -30,6 +43,13 @@ chomp($facility_name);
 my $service_files_base_dir="../gen/spool";
 my $service_files_dir="$service_files_base_dir/$facility_name/$service_name";
 
+# files used for computing a grace period
+my $MEMBER_STATE_FILE ="spool/$facility_name/ad_group_mu_ucn/memberState";
```

Comment:
I don't recommend storing 'last state' in the ``gen`` folder. I believe it can be wiped out by e.g. manually started gen or you lost it by renaming facility in Perun. Other services store such files in the ``send`` folder, since it is where the send script is located too.
In CRM service for ceitec we also store previous states in the ``logs/`` subfolder of the send. Its not the best, but reasonable solution if you would need to log/trace previous states and compare them.